Helios/Helios.csproj

```diff
@@ -18,6 +18,9 @@
   <PropertyGroup>
     <ApplicationIcon>Helios.ico</ApplicationIcon>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Development|x64'">
+    <LangVersion>latest</LangVersion>
```

Comment:
why is this here?

Reply:
The use of default value expressions caused my VS to barf because of the language level.  Happy for you to drop this, but I'll still need  to put this into my branch until I can upgrade.

Reply:
So you are saying you are on older Visual Studio or something and it is selecting a different language version?  That seems super weird since selecting 4.7.2 .NET should force language version 7?

Reply:
Unfortunately, target-typed default literals were not in 7.0, they came in at 7.1, so I have to set "latest minor version (latest)" to coax VS to use that language level.  Seems to be independent of the ,Net level.  I think you assert the need to build on 2019, but there are a couple of extensions for uControllers which do not work on 2019 so I am hanging on to 2017 until I can find an alternative solution.
Language level 7.**1** is the current minimum for Helios 1.6.